packages/cockpit/src/balance/operations/index.js

```diff
@@ -0,0 +1,45 @@
+import moment from 'moment'
+import fetchPayables from './payables'
+
+const operationsData = client => ({
+  count,
+  dates: {
+    end: endDate,
+    start: startDate,
+  },
+  page,
+  recipientId,
+  status,
+  type,
+}) => fetchPayables(client, {
+  count,
+  end_date: moment(endDate).valueOf(),
+  page,
+  recipientId,
```

Comment:
aqui acho que tem que ser `recipient_id` com underline pra api entender né?
```suggestion
  recipientId,
```